package.json

```diff
@@ -3,21 +3,110 @@
 	"displayName": "Payara VSCode",
 	"description": "Payara Tools for Visual Studio Code",
 	"version": "0.0.1",
+	"publisher": "Payara",
+	"repository": {
+		"type": "git",
+		"url": "https://github.com/payara/ecosystem-vscode-plugin"
+	},
+	"keywords": [
+		"Java",
+		"Jakarta EE",
+		"Payara"
+	],
 	"engines": {
 		"vscode": "^1.40.0"
 	},
 	"categories": [
 		"Other"
 	],
 	"activationEvents": [
-		"onCommand:extension.helloWorld"
+		"onCommand:payara.server.add",
+		"onCommand:payara.server.remove",
+		"onCommand:payara.server.rename",
+		"onCommand:payara.server.refresh"
 	],
-	"main": "./out/extension.js",
+	"main": "./out/main/extension.js",
 	"contributes": {
-		"commands": [{
-			"command": "extension.helloWorld",
-			"title": "Hello World"
-		}]
+		"commands": [
+			{
+				"command": "payara.server.add",
+				"title": "Add Payara Server",
+				"category": "Payara",
+				"icon": {
+					"light": "resources/light/add.svg",
+					"dark": "resources/dark/add.svg"
+				}
+			},
+			{
+				"command": "payara.server.remove",
+				"title": "Remove Payara Server",
+				"category": "Payara"
+			},
+			{
+				"command": "payara.server.rename",
+				"title": "Rename Payara Server",
+				"category": "Payara"
+			},
+			{
+				"command": "payara.server.refresh",
+				"title": "Refresh Payara Servers",
+				"category": "Payara",
+				"icon": {
+					"light": "resources/light/refresh.svg",
+					"dark": "resources/dark/refresh.svg"
+				}
+			},
+			{
+				"command": "payara.server.remove.context",
+				"title": "Remove"
+			},
+			{
+				"command": "payara.server.rename.context",
+				"title": "Rename"
+			}
+		],
+		"views": {
```

Comment:
I'm not personally a fan of this menu being in the explorer view (it's not related in any way to the open folders).

I'd personally favour this appearing in the `viewsContainers` contribution point, and then selecting and adding servers like how you manage debug configurations

Reply:
Added the Payara Server activity bar to `viewsContainers` but I would also recommend keeping Payara Server in explorer too, near to opened applications.